app/models/banana.rb

```diff
@@ -1,5 +1,5 @@
 class Banana < ActiveRecord::Base
-  def cut
+  def cut_cut
```

Comment:
__[method_definition_validator]__
* Lint/DefinitionValidator: cut is undefined. Did you mean? cut_cut at [app/models/banana.rb#L11](/actcat/sideci_rails_sandbox/blob/8f814ddecda9c71f1321383f85407d15ea95f368/app/models/banana.rb#L11)